src/compiler/compile/render_ssr/handlers/AwaitBlock.ts

```diff
@@ -13,7 +13,10 @@ export default function(node: AwaitBlock, renderer: Renderer, options: RenderOpt
 
 	renderer.add_expression(x`
 		function(__value) {
-			if (@is_promise(__value)) return ${pending};
+			if (@is_promise(__value)) {
+				__value.then(() => {}, () => {});
```

Comment:
should this be `.catch` instead of `.then`?  (i'm assuming you're right and I don't understand this, but just making sure)

Reply:
It's `.then(() => {}, () => {})`, which attaches a resolve and a reject handler. This seems microscopically safer than `.catch(() => {})`, because all `is_promise` has actually confirmed is that the value is an object with a `then` method. If it's a proper promise, we should be able to use `.catch` as well, but `.then` should also be equivalent.